Yoast/wordpress-seo
# The Events Calendar - Open Graph Schema

## Context


This plugin adds a the business logic of this plugin: https://de.wordpress.org/plugins/wpmyte/

The purpose of the code added is to provide a tighter integration of the schema code by 'The Events Calendar'. It transforms the schema code generated by TEC and puts it into a Schema_Piece.

## Summary


This PR can be summarized in the following changelog entry:

* Added Open Graph integration for `The Events Calendar`.

## Test instructions

This PR can be tested by following these steps:

* Install `The Events Calendar`
* Create a custom event
* Watch the month view for that event
* The Open Graph json object should now have a representation for that event
* Watch the single view for that event
* The Open Graph json object should now have a representation for that event

## Documentation
* [x] I have written (inline) documentation for this change.

## Quality assurance
